test/functional/test_framework/wallet.py

```diff
@@ -188,6 +188,41 @@ def send_to(self, *, from_node, scriptPubKey, amount, fee=1000):
         txid = self.sendrawtransaction(from_node=from_node, tx_hex=tx.serialize().hex())
         return txid, 1
 
+    def send_self_transfer_multi(self, from_node, utxos, **kwargs):
```

Comment:
nit:

```suggestion
    def send_self_transfer_multi(self, *, from_node, utxos, **kwargs):
```

Also, "utxos" can be renamed to "inputs" or "utxos_to_spend".